core/webviz/_header_element.py

```diff
@@ -0,0 +1,63 @@
+from six import iteritems
+
+
+class HeaderElement:
+    """
+    A HeaderElement describes one action taken to include a header element
+    to a Page.
+
+    :param tag: The tag of the header element, such as 'script' or 'link'.
+    :param attributes: Dictinary of attributes of the tag, such as `{'src':
+        'jquery.js'}`. Value strings can include the template value
+        `{root_folder}` which will be substituted with the path to the root of
+        the site.
+    :param content: The content of the text (inner html).
+    :param source_file: If the header element refers to a file, then the
```

Comment:
Isn't the source_file and target_file parameters unnecessary?
What will override the other, src, source_file or target_file?

Reply:
We also have the Html page element. 
Couldn't that be used for link and script?

Reply:
Unfortunately no as these resources also needs to be copied into the site and included in the header once, rather than whenever the page element is included.